apps/src/code-studio/peer_reviews/PeerReviewDashboard.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Spinner from '../pd/workshop_dashboard/components/spinner';
+import EscalatedSubmissions from './EscalatedSubmissions';
+
+const PeerReviewDashboard = React.createClass({
+  getInitialState() {
+    return {
+      escalatedSubmissions: undefined
+    };
+  },
+
+  componentDidMount() {
+    this.loadRequest = $.ajax({
+      method: 'GET',
+      url: '/api/v1/peer_review_submissions/index_escalated',
+      dataType: 'json'
+    }).done(data => {
+      this.setState({
+        escalatedSubmissions: data
+      });
+    });
+  },
+
+  renderEscalatedReviews() {
+    if (this.state.escalatedSubmissions) {
+      return (
+        <EscalatedSubmissions
+          submissions={this.state.escalatedSubmissions}
+        />
+      );
+    } else {
+      return (
+        <Spinner/>
+      );
+    }
+  },
+
+  render() {
+    return (
+      <div>
+        {this.renderEscalatedReviews()}
+      </div>
+    );
+  }
```

Comment:
Like above, why encapsulate this in a method? And why wrap in a div? Why not follow the existing pattern of returning early with the spinner, and otherwise rendering the main component in `render` directly? See [this](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/workshop_dashboard/workshop.jsx#L687-L702) and [this](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/workshop_dashboard/components/workshop_table_loader.jsx#L95-L102)